docs/data/xrootd/install-standalone.md

```diff
@@ -1,2 +1,182 @@
+
+An example of an authorization file for read only of store by CMS would be:
+
+```file
+g /cms /store lr
+```
+
+#### LCMAPS authorization
+
+To configure the LCMAPS authentication please follow the documentation
+[here](/data/install-xrootd/#security-option-3-xrootd-lcmaps-authorization)
+
+### Optional Configuration
+
+The following configuration steps are optional and will likely not be required for setting up a small site.
+If you do not need any of the following special configurations, skip to
+[the section on using XRootD](#using-xrootd).
+
+#### Enabling HTTP support
+
+In order to enable XRootD HTTP support please follow the instructions
+[here](/data/install-xrootd/#optional-enabling-xrootd-over-http)
+
+#### Enabling Hadoop support
+
+For documentation on how to export your Hadoop storage using XRootD please see
+[this documentation](/data/install-xrootd/#optional-adding-hadoop-support-to-xrootd)
+
+#### Enabling CMS TFC support (CMS sites only)
+
+For CMS users, there is a package available to integrate rule-based name lookup using a `storage.xml` file.
+See [this documentation](/data/install-xrootd/#optional-adding-cms-tfc-support-to-xrootd-cms-sites-only).
+
+Using XRootD
+------------
+In order to start your XRootD server you need to run
+
+``` console
+root@xrootd-server # systemctl start xrootd@%RED%server%ENDCOLOR%
+```
+
+As a reminder, here are common service commands (all run as `root`):
+
+| To …                                        | On EL 6, run the command…    | On EL 7, run the command…        |
+|:--------------------------------------------|:-----------------------------|:---------------------------------|
+| Start a service                             | `service SERVICE-NAME start` | `systemctl start SERVICE-NAME`   |
+| Stop a service                              | `service SERVICE-NAME stop`  | `systemctl start SERVICE-NAME`   |
+| Enable a service to start during boot       | `chkconfig SERVICE-NAME on`  | `systemctl enable SERVICE-NAME`  |
+| Disable a service from starting during boot | `chkconfig SERVICE-NAME off` | `systemctl disable SERVICE-NAME` |
+
+Validating XRootD
+-----------------
+
+To validate an XRootD installation, perform the following verification steps:
+
+1. Verify file transfer over the XRootD protocol using XRootD client tools:
+
+    1. Install the client tools:
+
+            :::console
+            root@xrootd-server # yum install xrootd-client
+
+    1. Copy a file to a directory for which you have write access:
+
+            :::console
+            root@xrootd-server # xrdcp /bin/sh root://localhost:1094//tmp/first_test
+            [xrootd] Total 0.76 MB  [====================] 100.00 % [inf MB/s]
+
+    1. Verify that the file has been copied over:
+
+            :::console
+            root@xrootd-server # ls -l /tmp/first_test
+            -rw-r--r-- 1 xrootd xrootd 801512 Apr 11 10:48 /tmp/first_test
```

Comment:
Does this actually work if `all.export` is set to `/storage/`? I thought if they set that, that the file would actually live in `/storage/tmp/first_test`.

Same thing with the HTTP validation steps below.

Reply:
Yes `/tmp` is always allowed. I can change it to `/storage/firs_test`

Reply:
Edgar explained this to me on Slack. Without `localroot` configuration, XRootD exposes the filesystem so `/tmp` actually does map to `/tmp`